Lib/test/test_complex.py

```diff
@@ -269,6 +269,33 @@ def test_pow(self):
                     except OverflowError:
                         pass
 
+        # Check that small integer exponents are handled identically
```

Comment:
It'd be nice to make this its own test method rather than just appending more code to the existing `test_pow()`.

Reply:
Agreed, and done.